### DISCUSSION TITLE
[Projects Beta] enable full screen mode like in the older projects version

### DISCUSSION THREAD

**Discussion by xh3b4sd** [community/community#10033] [Projects and Issues]
I would like to to hide or remove all the complexity from the view when looking at a project. There are 4 layers of navigation and none of these are relevant for people wanting to have an overview of the project board. I think the interface could be way more clean and the design way less distracting for the eyes. 

<img width="792" alt="Screenshot 2022-01-12 at 13 34 18" src="https://user-images.githubusercontent.com/552769/149141197-43c2e1a6-8a1d-44db-82b5-7799c5ddc174.png">

**Comment by knoan** [comment]
Make only the highlighted area visibile in full screen.

![full-screen](https://user-images.githubusercontent.com/6105462/152486452-ce3a3bcb-500d-4709-8164-8f837dea8043.jpg)

**Comment by knoan** [comment]
To further optimize vertical screen real estate (which is always scarce) move card annotation inline with the label
![inline](https://user-images.githubusercontent.com/6105462/152486603-95596ccd-0711-454b-9697-f52c9a90948e.png)

**Comment by wasciutto** [comment]
Would like to bump this issue - The new projects boards are barely get any real-estate on a laptop screen, and even on a larger screen the 4 layers of bars above the boards are hard to look at. Makes it difficult for me to move away from classic boards.

**Comment by RalfJung** [comment]
Has the new top bar now been landed without opt-out and without fullscreen project support? Congrats Github, you made my screen 30% less useful. :(

I guess now we know how much GH cares for the feedback that they are asking for in their betas.

**Comment by johnml1135** [comment]
A few potential improvements:
* Put the name of the project (and the options 
![image](https://github.com/community/community/assets/13733556/892640b9-503f-45e6-bed4-dba4c1dcbf02)
in-line with the project tabs
* Have at least the top GitHub bar shrink away when scrolling down
These should be able to be done without reworking the main UI's for roadmaps, Kanban and tables while hiding unimportant things when looking at issues (do I really want to create a new repository when looking at my project?)
